pkg/defaultRules/defaultRules.yaml

```diff
@@ -1548,3 +1548,273 @@ rules:
       anyOf:
         - $ref: "#/definitions/metadataNamePattern"
         - $ref: "#/definitions/metadataGenerateNamePattern"
+  - id: 55
+    name: Ensure container probes have an initial delay configured
+    uniqueName: CONTAINERS_INCORRECT_INITIALDELAYSECONDS_VALUE
+    enabledByDefault: true
+    documentationUrl: "https://hub.datree.io/built-in-rules/ensure-initial-probe-delay"
+    messageOnFailure: "Incorrect value for key `initialDelaySeconds` - set to prevent unwanted default behavior"
```

Comment:
"set to prevent unwanted default behavior" - this is too generic. let's try to make it more informative and customized for each rule.

Reply:
> it's not an incorrect value, it's a missing value (applied to all rules)

@eyarz 
The rule will fail not only if the value is missing, but also if it is incorrect. So "missing" is a particular case of "incorrect", but not the other way around. WDYT?

Reply:
good point. I agree.
the best option will be to break this rule into two (one for "exists" and for "correct value"), but I feel it will be too spammy/redundant to the users.

Reply:
Here are 2 examples using "Incorrect" when the value could also be missing:
https://hub.datree.io/built-in-rules/ensure-configmap-is-recognized-by-argocd
https://hub.datree.io/built-in-rules/ensure-read-only-filesystem

Looks like we're already using this behavior:)